scripts/zones/Misareaux_Coast/Zone.lua

```diff
@@ -3,32 +3,53 @@
 -- Zone: Misareaux_Coast (25)
 --
 -----------------------------------
-local ID = require("scripts/zones/Misareaux_Coast/IDs")
 require("scripts/globals/conquest")
 require("scripts/globals/helm")
+require("scripts/zones/Misareaux_Coast/IDs")
 -----------------------------------
 
 function onInitialize(zone)
     tpz.helm.initZone(zone, tpz.helm.type.LOGGING)
-end;
+end
 
 function onConquestUpdate(zone, updatetype)
     tpz.conq.onConquestUpdate(zone, updatetype)
-end;
+end
 
 function onZoneIn(player,prevZone)
-    local cs = -1;
+    local cs = -1
     if (player:getXPos() == 0 and player:getYPos() == 0 and player:getZPos() == 0) then
-        player:setPos(567.624,-20,280.775,120);
+        player:setPos(567.624,-20,280.775,120)
     end
-    return cs;
-end;
+    return cs
+end
 
 function onRegionEnter(player,region)
-end;
+end
+
+function onGameHour(zone)
+    local baited = GetServerVariable("[Ziphius]Bait Trap")
+    local ZIPHIUS_QM_BASE  = 16879919
+    if (VanadielHour() == 22) then  -- Spawn traps for Ziphius
```

Comment:
Hours check should maybe have >= and <= times so if the sever is restarted they will still work for the current day